### PR TITLE
newsletter October 2024

### LINKED ISSUE #2148: Adding Newsletter to Julia Language Blog
@Krastanov some of us have been talking about potentially adding your monthly newsletter as a blogpost to the Julia Language Blog.

Is that something you'd be interested in? I don't think it would involve too much additional effort from you. We'd probably like the blog PR to be made 1-2 days before the actual newsletter is posted to give us time to edit it. But otherwiseI think it's a good idea. 

Of course we'd still post the same post or link to the blog on Discourse each month for people to comment on etc.

What are your thoughts? Also anybody else paying attention to this repo what are your thoughts? Our goal is to keep the blog low noise, so no more than 2-3 posts per month at peak. But currently there are vanishingly few posts throughout the year, and this is one way to encourage community growth and advertise packages to the world.

### PR DESCRIPTION
**Pull request by Krastanov**
a draft from https://docs.google.com/document/d/1Np1deH_W1o0EO7_tZkScveCLA0XiYg538ceUJs0bFpE/edit

fixes #2148

@rayegun

### CODE REVIEW

File: blog/2024/10/this-month-in-julia-world.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+A monthly newsletter, mostly on julia internals, digestible for casual observers. A biased, incomplete, editorialized list of what I found interesting this month, with contributions from the community.
+
+“Internals” Fora and Core Repos (Slack/Zulip/Discourse/Github):
+
+* [Well documented tips on how to minimize latency](https://docs.julialang.org/en/v1.12-dev/manual/performance-tips/#Execution-latency,-package-loading-and-package-precompiling-time) in your packages are now organized in a new section in the “Performance Tips” docs
+* The trace-compile capabilities provide a convenient way to track what the compiler is working on as you ecnounter new methods during runtime. It is extremely helpful when you want to figure out what precompile statements to add to your packages. A few quality-of-life improvements are coming up, with [better colored output for the traces](https://github.com/JuliaLang/julia/pull/55763).
+* Relatedly, recently we got a special-case of trace-compile, [the new trace-dispatch](https://github.com/JuliaLang/julia/pull/55848) which prints precompile statements only for dynamically encountered methods.
```
Comment (IanButterworth) [inline blog/2024/10/this-month-in-julia-world.md:13]: This is likely to be merged soon to https://github.com/JuliaLang/julia/pull/55915

Comment (Krastanov) [reply blog/2024/10/this-month-in-julia-world.md:13]: this is really cool! And now it is included

**Review by rayegun — APPROVED** [review]
I think this is good to go, but I'll wait for one more person to review before merging!

**Comment by Krastanov** [comment]
Thanks! Now posted on discourse https://discourse.julialang.org/t/this-month-in-julia-world-2024-09/120974

**Comment by IanButterworth** [comment]
Great. I see no issues raised in discourse so I'll merge. Thanks as ever!

**Pull request merged**